english/marshmallow.md

```diff
@@ -81,6 +81,8 @@ The Chief Marshmallow also demands that you watch [Sonicsgate](https://www.youtu
 
 Or a Wednesday...Thursday...maybe even a Friday, but definetely not a Saturday.
 
+Or a [Wednesday...](make_changes_GitHub/make_changes_GitHub.md) [Thursday...](doug-adams/arthur-dent.md) maybe even a Friday, but definetely not a Saturday.
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.